manager/manifests/cluster-autoscaler.yaml

```diff
@@ -0,0 +1,166 @@
+# Copyright 2019 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Maybe link to where you got this YAML from (including the version e.g. via tag)? For example, I'll link to https://github.com/kubernetes-incubator/metrics-server/tree/v0.3.3/deploy/1.8+ in mine